src/main/kotlin/org/jetbrains/intellij/dependency/PluginDependencyManager.kt

```diff
@@ -106,7 +99,8 @@ open class PluginDependencyManager @Inject constructor(
     private fun registerRepositoryIfNeeded(project: Project, plugin: PluginDependency) {
         if (ivyArtifactRepository == null) {
             ivyArtifactRepository = project.repositories.ivy {
-                it.ivyPattern("$cacheDirectoryPath/[organisation]/[module]-[revision].[ext]") // ivy xml
+                val ivyFileSuffix = plugin.getFqn().substring("${plugin.id}-${plugin.version}".length)
```

Comment:
I think it won't work. for `plugin-123-2.xml` prefix would be equal to `-2.xml` and according to pattern the following file will be expected: `$cacheDirectoryPath/[organisation]/[module]-[revision]-2.xml.xml`

Reply:
`plugin.getFqn()` does not have `.xml` as a suffix:
![pluginFqn](https://user-images.githubusercontent.com/4990763/123935561-044a4080-d99d-11eb-9f75-9dfe11088cfb.png)